src/ucp/core/ucp_mm.c

```diff
@@ -1595,7 +1600,7 @@ ucp_memh_import_slow(ucp_context_h context, ucs_rcache_t *existing_rcache,
 
             ucs_rcache_set_default_params(&rcache_params);
 
-            status = ucp_mem_rcache_create(context, rcache_name, &rcache,
+            status = ucp_mem_rcache_create(context, rcache_name, &rcache, 0,
```

Comment:
why need to avoid events for imported memory in this PR, does it solve some CI failure?

Reply:
`shm_ib/test_ucp_mmap_export.reg_export_and_reimport/2` started to fail on `swx-rain03-bf1`.
isn't it correct fix? this rcache caching memory registrations of foreign host, why should we invalidate basing on memory events happening in local host?